frontend/app/src/components/StreamlitDialog/StreamlitDialog.tsx

```diff
@@ -246,65 +240,6 @@ function clearCacheDialog(props: ClearCacheProps): ReactElement {
   )
 }
 
-interface RerunScriptProps {
```

Comment:
note: I would have had to update the style values for the Rerun Dialog and then realized that it does not seem to be used anymore. 
@kmcgrady could you have a look for sanity checking this? The Rerun Dialog seems to be very old, but does not seem to be used anywhere nowadays